model/preference.go

```diff
@@ -13,6 +13,24 @@ import (
 )
 
 const (
+	PREFERENCE_CATEGORY_DIRECT_CHANNEL_SHOW = "direct_channel_show"
+	PREFERENCE_CATEGORY_GROUP_CHANNEL_SHOW  = "group_channel_show"
+	PREFERENCE_CATEGORY_TUTORIAL_STEPS      = "tutorial_step"
+	PREFERENCE_CATEGORY_ADVANCED_SETTINGS   = "advanced_settings"
+	PREFERENCE_CATEGORY_FLAGGED_POST        = "flagged_post"
+	PREFERENCE_CATEGORY_FAVORITE_CHANNEL    = "favorite_channel"
+	PREFERENCE_CATEGORY_SIDEBAR_SETTINGS    = "sidebar_settings"
+
+	PREFERENCE_CATEGORY_DISPLAY_SETTINGS      = "display_settings"
+	PREFERENCE_NAME_COLLAPSED_THREADS_ENABLED = "collapsed_reply_threads"
+	PREFERENCE_NAME_CHANNEL_DISPLAY_MODE      = "channel_display_mode"
+	PREFERENCE_NAME_COLLAPSE_SETTING          = "collapse_previews"
+	PREFERENCE_NAME_MESSAGE_DISPLAY           = "message_display"
+	PREFERENCE_NAME_NAME_FORMAT               = "name_format"
+	PREFERENCE_NAME_USE_MILITARY_TIME         = "use_military_time"
+	PREFERENCE_RECOMMENDED_NEXT_STEPS         = "recommended_next_steps"
+
+	PREFERENCE_CATEGORY_THEME           = "theme"
```

Comment:
Are all these new preference const related?

Also, can we keep the same camel case for consistency?

Reply:
I only added `PREFERENCE_RECOMMENDED_NEXT_STEPS ` but not sure why github is showing all those as additions I made.

Reply:
The issue was the way I resolved conflicts. Fixed now.